phpBB/phpbb/files/types/remote.php

```diff
@@ -106,108 +99,55 @@ protected function remote_upload($upload_url)
 
 		$url['path'] = implode('', $url['path']);
 		$upload_ary['name'] = utf8_basename($url['path']) . (($ext) ? '.' . $ext : '');
-		$filesize = 0;
 
 		$remote_max_filesize = $this->get_max_file_size();
 
-		$errno = 0;
-		$errstr = '';
+		$guzzle_options = [
+			'timeout'			=> $this->upload->upload_timeout,
+			'connect_timeout'	=> $this->upload->upload_timeout,
+			'verify'			=> !empty($this->config['remote_upload_verify']),
```

Comment:
If you cast the setting to boolean it can be either false if you have it set to false, 0, etc. or true if you have anything like 1 or true set. This will result in exactly the same.